webapp/components/admin_console/rate_settings.jsx

```diff
@@ -120,7 +120,7 @@ export default class RateSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.rate.remoteTitle'
-                            defaultMessage='Vary By Remote Address: '
+                            defaultMessage='Limit by Remote Address: '
```

Comment:
Check with dev on this and update help text? Not clear what this does and how it should be named.